src/resources/lang.json

```diff
@@ -73,7 +73,7 @@
     {
       "en": "My places",
       "de": "Meine Orte",
-      "it": "Miei posti",
+      "it": "Siti",
```

Comment:
This is not correct. The translation is missing "my".

Reply:
But this is what Milena wanted? 

> The terms :  "Siti" instead of "Miei Posti " ;"Sito" instead of "Posto".